test/xpu_api/utilities/function.objects/refwrap/refwrap.helpers/ref_1.pass.cpp

```diff
@@ -0,0 +1,65 @@
+// -*- C++ -*-
+//===----------------------------------------------------------------------===//
+//
+// Copyright (C) Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+// This file incorporates work covered by the following copyright and permission
+// notice:
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+//
+//===----------------------------------------------------------------------===//
+
+// <functional>
+
+// reference_wrapper
+
+// template <ObjectType T> reference_wrapper<T> ref(T& t);
+
+#include "support/test_config.h"
+
+#include <oneapi/dpl/functional>
+
+#include "support/utils.h"
+
+#if TEST_DPCPP_BACKEND_PRESENT
+#    include "support/counting_predicates.hpp"
```

Comment:
Counting predicates are not used in this test

Reply:
Fixed

Reply:
fixed